packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue

```diff
@@ -74,7 +74,10 @@ const isAnnotationEnabled = computed(
 );
 
 const runningExecutionsCount = computed(() => {
-	return props.executions.filter((execution) => execution.status === 'running').length;
+	return props.executions.filter(
+		(execution) =>
+			execution.status === 'running' && ['webhook', 'trigger'].includes(execution.mode),
```

Comment:
Important - I don't understand why we need to also check for the mode now? Same below.